module-os/board/linux/FreeRTOSConfig.h

```diff
@@ -104,12 +103,9 @@ extern "C"
 #define configTIMER_TASK_STACK_DEPTH (1024)
 
 /* Define to trap errors during development. */
-#define configASSERT(x)                                                                                                \
-    if ((x) == 0) {                                                                                                    \
-        LOG_FATAL("Assertion: %s", x);                                                                                 \
-        while (1)                                                                                                      \
-            ;                                                                                                          \
-    }
+#ifndef NDEBUG
+#define configASSERT(x) assert(x)
```

Comment:
while(1) was to reboot the os via watchdog and LOG_FATAL to set in the log file why the OS rebooted. Wasn't it?

Reply:
This does not apply as this is Linux-specific `FreeRTOSConfig.h`.

Reply:
Apart from that, I didn't change the way configASSERT works. It's defined exactly as it was before.